activesupport/lib/active_support/testing/assertions.rb

```diff
@@ -32,6 +32,7 @@ def assert_not(object, message = nil)
       #   end
       def assert_nothing_raised
         yield
+        assert true
```

Comment:
I could probably use `self.assertions += 1` but it seems a little to much knowledge about minitest internals 
I would prefer having a public method like `increment_assertions` exposed by minitest 🤔